### PR TITLE
add gdunderhood, please

### PR DESCRIPTION
**Pull request by meenauh**

### CODE REVIEW

**Comment by iamstarkov** [comment]
lets wait two weeks

**Comment by iamstarkov** [comment]
its a rule to wait one month, to find out will account be abandoned or not. yours one already passed two weeks. So two weeks left

**Comment by meenauh** [comment]
it's actually the end of the 4th week (http://gdunderhood.ru/ — 4th author is live right now).

**Comment by iamstarkov** [comment]
pls, rebase and ill merge it!

**Comment by meenauh** [comment]
@iamstarkov done

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Inspired by the [awesome][awesome-url] list thing.
 * [@abroadunderhood](https://twitter.com/abroadunderhood) — relocation to other country
 * [@spbunderhood](https://twitter.com/spbunderhood) — about Saint Petersburg
 * [@sexsecrethood](https://twitter.com/sexsecrethood) — nsfw
+* [@ceounderhood](https://twitter.com/ceounderhood) — CEO and management
```
Comment (iamstarkov) [inline README.md:19]: why its here?

Comment (meenauh) [inline README.md:19]: it's in master. I rebased on your master branch. dunno why it's "added"

Comment (iamstarkov) [inline README.md:19]: if its in the master, then it should not be in you changes

**Pull request closed without merging**